vm/vmStmt.go

```diff
@@ -488,6 +488,85 @@ func runSingleStmt(stmt ast.Stmt, env *Env, ctx context.Context) (reflect.Value,
 		env.defineGlobalValue(stmt.Name, reflect.ValueOf(newenv))
 		return rv, nil
 
+	// SelectStmt
+	case *ast.SelectStmt:
+		newenv := env.NewEnv()
+		body := stmt.Body.(*ast.SelectBodyStmt)
+		letsStmts := []*ast.LetsStmt{nil}
```

Comment:
Is `letsStmts` or `bodies` really needed? Can one use `chosen` later to access wanted LetsStmt and Stmt?

Reply:
I don't see a good way of doing this without the letsStmts and bodies.